### DISCUSSION TITLE
Blog Post: Getting Familiar with the Workspace

### DISCUSSION THREAD

**Discussion by altugbakan** [stellar/sorobanathon#67] [General]
Hello!

I have written a blog post on the workspace used for developing Soroban contracts on GitPod workspace. I am planning to write more detailed and technical posts about the quests, and even some other custom contracts on Soroban if this kind of content gets enjoyed by fellow developers. You can find my blogpost [here](https://dev.to/altug/soroban-quest-getting-familiar-with-the-workspace-17h9) on dev.to.

This is my first time writing a technical tutorial, so please feel free to share some criticism so that I can improve my content more.

Please also don't hesitate to share what might be missing, what else might I include, and even some ideas for a new post. I hope you like it!

**Comment by wildework** [comment]
Nice! I haven't done any Stellar Quests before so this was insightful, thanks.

**Comment by achallagundla** [comment]
Heya @altugbakan  ,
Congratulations! Your contribution to Sorobanathon: First Light has been accepted! 🎉
Your submission may qualify for a reward of 1500 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions. (You can also message me on discord @ anuxhya)

All the best,
Anuhya
